apps/institution-service/tasks/loadInstitutionsFromJson.ts

```diff
@@ -0,0 +1,114 @@
+import fs from "fs";
+import path from "path";
+import db from "../config/database";
+import { Institution } from "../models/institution";
+import { Provider } from "../models/provider";
+
+export interface CachedInstitution {
+  name: string;
+  keywords: string | null;
+  logo: string;
+  url: string;
+  ucp_id: string;
+  is_test_bank: boolean | false;
```

Comment:
I've never seen `boolean | false` before. What does this do?

Reply:
I was thinking maybe it's a way to set a default value, but I can't find any documentation to support that. I don't think using `boolean | false` is necessary here. `boolean` should suffice, unless I'm not understanding the meaning here.

Reply:
Yeah that was an accident, it was supposed to be boolean | null. I'll fix it.

Reply:
Actually I don't even need the `| null` at all anymore.